include/merlin_hashtable.cuh

```diff
@@ -150,13 +163,15 @@ static constexpr auto& thrust_par = thrust::cuda::par;
  *
  */
 template <typename K, typename V, typename S = uint64_t,
-          typename ArchTag = Sm80>
+          int Strategy = EvictStrategy::kLru, typename ArchTag = Sm80>
```

Comment:
Since kEpochLru and kEpochLfu need 64bit score. Need to check size of `typename S` in case.

Reply:
Yes, I added the type check for S : https://github.com/rhdong/HierarchicalKV/blob/rhdong/epoch-lru/include/merlin_hashtable.cuh#L194